src/coreclr/src/vm/readytoruninfo.cpp

```diff
@@ -60,10 +70,7 @@ MethodDesc * ReadyToRunInfo::GetMethodDescForEntryPoint(PCODE entryPoint)
         return NULL;
```

Comment:
I think this can be in `TryGetMethodDescForEntryPoint` as well. The comment talks about PtrHashMap. Since the PtrHashMap logic was moved into TryGetMethodDescForEntryPoint, it looks odd now here in isolation.

Reply:
Moved in 3rd commit.